docker-compose.yml

```diff
@@ -19,4 +19,13 @@ services:
     image: redis:6.2.6-bullseye
     network_mode: bridge
     ports: 
-      - "6379:6379"
\ No newline at end of file
+      - "6379:6379"
+  db:
+    image: postgres:13-alpine
+    network_mode: bridge
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_DB: posthog
```

Comment:
Are we sure we want to leverage the same name as we do for the main stack?

Nevertheless as per my comment above I'm pro getting rid of the Postgres dependency